plugins/Referrers/lang/en.json

```diff
@@ -18,6 +18,8 @@
         "CampaignGroupHelp": "This parameter is used to track a Campaign Group to identify a group or audience. Campaign groups are helpful for tracking the overall performance of multiple campaigns with similar goals.",
         "CampaignPlacement": "Campaign Placement",
         "CampaignPlacementHelp": "Used to describe the advertising placement or position, for example an entire website, specific pages from a website, an individual ad unit positioned on a single page, a video, a mobile phone app, and more.",
+        "CampaignId": "Campaign ID",
+        "CampaignIdHelp": "This parameter is used to track a Campaign ID: an identifier which can be a number or any string of data up to 100 characters.",
```

Comment:
I think this might sound better as: "a Campaign ID: a unique identifier for the command consisting of at most 100 numbers, letters or symbols.", what do you think?

Reply:
Updated 👍

Reply:
@tsteur that was the text we are using here as well: https://matomo.org/docs/tracking-campaigns-url-builder/
Maybe we should update that one then, too?

Reply:
updated it, page might be cached though for a bit